extensions/mqtt/controllerservice/MQTTControllerService.h

```diff
@@ -293,8 +289,8 @@ class MQTTControllerService : public core::controller::ControllerService {
   MQTTClient client_;
   std::string uri_;
   std::string topic_;
-  int64_t keepAliveInterval_;
-  int64_t connectionTimeOut_;
+  std::chrono::milliseconds keepAliveInterval_{0};
+  std::chrono::milliseconds connectionTimeOut_{0};
```

Comment:
good idea, changed it in https://github.com/apache/nifi-minifi-cpp/pull/1225/commits/9f0c3752c6a328e005bb22a566eceb406c7fe348

Reply:
I see the changes in other files, but this particular one is not changed in the commit

Reply:
you are right I do seem to have skipped this, probably because it is not used anywhere :confused: 
i've looked at MQTTControllerService and it has a bunch of properties and matching members that are not used anywhere as far as I could tell.
from BrokerURL, ClientID, CleanSession, UserName, Password, KeepLiveInterval, ConnectionTimeout, QOS Topic, SecurityProtocol only BrokerURL and ClientID is used in onEnable

Am i missing something ?

Reply:
it is indeed unused, so it's probably best to remove it

Reply:
I will create a separate ticket and investigate, because the MQTTControllerService::reconnect relies on some of these uninitialized members and the AbstractMQTTProcessor and MQTTControllerService have a lot of duplicate codes.
I wouldnt be suprised if there are some bugs here or there.

Anyways I fixed the timeout typos throughout the code in https://github.com/apache/nifi-minifi-cpp/pull/1225/commits/63a26de4232196b77a7a366ad95f99a3a62fc598